src/main.ts

```diff
@@ -45,8 +98,9 @@ async function bootstrap() {
   const port = app.get<ConfigService>(ConfigService).get<number>('server.port');
   await app.listen(port);
 
-  logger.log({ message: 'server started 🚀', port, url: `http://localhost:${port}/api/v1` });
+  logger.log({ message: 'Server started 🚀', port, url: `http://localhost:${port}/api/v1` });
```

Comment:
You should not be touching shared files that do not affect your implementation

Reply:
okay will do work on it